source/api.py

```diff
@@ -317,8 +317,15 @@ def getStatusBar():
 	if not location:
 		return None
 	left, top, width, height = location
-	bottom = top + height - 1
-	obj = getDesktopObject().objectFromPoint(left, bottom)
+	x=1
+	while x<12:
+		bottom = top + height - x
+		if left<0:
+			left=0

```

Comment:
And for non maximised, does bottom=top+height-11 work or break? I vaguely recall you saying it broke for non maximised.